environments/base/manifests/thanos-query-frontend-deployment.yaml

```diff
@@ -47,34 +47,14 @@ spec:
         - --http-address=0.0.0.0:9090
         - --query-frontend.downstream-url=http://observatorium-xyz-thanos-query.observatorium.svc.cluster.local.:9090
         - --query-range.split-interval=24h
-        - --labels.split-interval=24h
         - --query-range.max-retries-per-request=0
-        - --labels.max-retries-per-request=0
         - --query-frontend.log-queries-longer-than=5s
         - |-
           --query-range.response-cache-config="config":
-            "addresses":
-            - "dnssrv+_client._tcp.observatorium-xyz-thanos-query-frontend-memcached.observatorium.svc"
-            "dns_provider_update_interval": "10s"
-            "max_async_buffer_size": 10000
-            "max_async_concurrency": 20
-            "max_get_multi_batch_size": 0
-            "max_get_multi_concurrency": 100
-            "max_idle_connections": 100
-            "timeout": "500ms"
-          "type": "memcached"
-        - |-
-          --labels.response-cache-config="config":
-            "addresses":
-            - "dnssrv+_client._tcp.observatorium-xyz-thanos-query-frontend-memcached.observatorium.svc"
-            "dns_provider_update_interval": "10s"
-            "max_async_buffer_size": 10000
-            "max_async_concurrency": 20
-            "max_get_multi_batch_size": 0
-            "max_get_multi_concurrency": 100
-            "max_idle_connections": 100
-            "timeout": "500ms"
-          "type": "memcached"
```

Comment:
This is weird. I don't see `query-frontend` related part updated in this pr, but why the yaml was changed?

Reply:
Oh, I guess you need to re-generate the manifests since the E2E is failing

Reply:
Maybe a rebase error? What's exactly missing?

Reply:
Aha CI also failing because of this